packages/payment-experience-api/src/openapi.yaml

```diff
@@ -414,4 +552,50 @@ paths:
                 $ref: '#/components/schemas/Payment'
         '404':
           $ref: '#/components/responses/NotFound'
-
+  /refund/instant/{orderId}:
+    get:
+      description: Create a refund for single order, confirm it and create a payment - refunds all order items.
```

Comment:
Create a payment-> create a refund payment

Reply:
Good point, I'll update the text.